testsuite/httpx/auth.py

```diff
@@ -23,12 +23,12 @@ def token(self):
         return self._token
 
     def _add_credentials(self, request: Request, token):
-        if self.location == 'authorization':
-            request.headers['Authorization'] = f"Bearer {token}"
-        elif self.location == 'headers':
-            request.headers['access_token'] = token
-        elif self.location == 'query':
-            request.url = URL(request.url, params={'access_token': token})
+        if self.location == "authorization":
```

Comment:
Big plus of Black is that it is consistent on using quotation marks and apostrophes.